controller/src/main/java/com/pilot/controller/HelloController.java

```diff
@@ -0,0 +1,23 @@
+package com.pilot.controller;
+
+
+import com.pilot.service.HelloService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+
+    private HelloService helloService;
+
+    public HelloController(HelloService helloService){
```

Comment:
We don't have to write this constructor if we can make use of lombok. `@RequiredArgsConstructor` from lombok will generate constructor for all final fields. would suggest you make `helloService` declaration final and add `@RequiredArgsConstructor`